### PR TITLE
winning-tickets-concepts

### PR DESCRIPTION
**Pull request by ArcAster**
v1 doc

### CODE REVIEW

File: docs/video-miners/core-concepts/winning-ticket-concepts.md [hunk 1]
```diff
@@ -0,0 +1,40 @@
+# Ticket EV for Orchestrators
+
```
Comment (hnjolles1) [inline docs/video-miners/core-concepts/winning-ticket-concepts.md:10]: See most recent commit for introductory paragraph.

**Review by hnjolles1 — APPROVED** [review]

File: docs/video-miners/core-concepts/winning-ticket-concepts.md [hunk 2]
```diff
@@ -0,0 +1,79 @@
...
+***Related documentation*** [Earnings](https://docs.livepeer.org/video-miners/core-concepts/earnings) and [Payments](https://docs.livepeer.org/video-miners/core-concepts/payments).
+
+## What is a ticket?
+
+Broadcasters using the Livepeer network attach a “ticket” to each segment of a video they want transcoded. A ticket can be thought of as a form of work completed for a given video stream a broadcaster is emitting to the Livepeer network.  The final number of tickets associated with a given stream is a product of the fees proposed by the orchestrator for doing the work and the total length of the video processed.
+
+A ticket is comprised of the following attributes:
+
+- **recipient**: ETH address of recipient (orchestrator completing transcoding work)
+- **faceValue**: The ETH value of the ticket payable to the recipient (orchestrator) if the ticket wins
```
Comment (ecmulli) [inline docs/video-miners/core-concepts/winning-ticket-concepts.md:20]: how is this value derived? its set by the broadcaster but how is it set for each ticket?
how is this related to the `maxFaceValue` parameter set by orchestrators?

File: docs/video-miners/core-concepts/winning-ticket-concepts.md [hunk 3]
```diff
@@ -0,0 +1,79 @@
...
+
+## How Winning Tickets Are Picked
+
+Picking tickets isn’t as simple as it might seem, we use an implementation based on [probabilistic micropayments](https://medium.com/livepeer-blog/streamflow-probabilistic-micropayments-f3a647672462).  At a high level, each ticket could be thought of as a “lottery” ticket of sorts.  But the raw implementation is a bit more complex than it might seem.
+
+Orchestrators can set a configuration value called ticket expected value or “EV” which defines the value of work an orchestrator is willing to do before requiring a ticket.  By default this value is set to 1000 gwei, which translates to an orchestrator willing to complete 1000 gwei worth of transcoding (based on their pixel pricing configuration) before requiring a ticket.
+
+Winning tickets are selected based on a ticket’s win probability (winProb) and a random value produced by hashing the broadcaster’s random number with a random number from the orchestrator linked to the commitment included in the ticket (recipientHash).  This process is completed at the end of each “round” on the Livepeer network.
```
Comment (ecmulli) [inline docs/video-miners/core-concepts/winning-ticket-concepts.md:35]: Could you elaborate on this "hashing" function? What is it checking?

File: docs/video-miners/core-concepts/winning-ticket-concepts.md [hunk 4]
```diff
@@ -0,0 +1,40 @@
+# Ticket EV for Orchestrators
+
+## What is a ticket?
+
+Broadcasters using the Livepeer network attach a “ticket” to each segment of a video they want transcoded. A ticket can be thought of as a form of work completed for a given video stream a broadcaster is emitting to the Livepeer network.  The final number of tickets associated with a given stream is a product of the fees proposed by the orchestrator for doing the work and the total length of the video processed.
+
```
[the remark this answers is not in the document]
Comment (ArcAster) [reply docs/video-miners/core-concepts/winning-ticket-concepts.md:16]: Updated

File: docs/video-miners/core-concepts/winning-ticket-concepts.md [hunk 5]
```diff
@@ -0,0 +1,40 @@
...
+If an orchestrator receives a winning ticket, the ticket can submit the ticket along with their recipientRandomHash to a smart contract which will verify the hash and attest that the ticket indeed won.  Given a pair of broadcaster and orchestrator (payor and payee) neither party can rig the system as the random value used to select winning ticket can only be seen by one party during the process of generating the ticket as the payee is forced to commit it’s random value prior to seeing the payer’s random number.
+
+## How Often *Should* I win
+
+Although each “win” is awarded randomly, the probabilistic in “probabilistic micropayments” ensures that even with adverse impact from factors like gas prices or fluctuation in network activity - at a high level payments follow a linear distribution trend.
+
+- how long it takes to approach certainty mathematically with payments
+    - within 25% of where you should in theory be
+    - std dv of where you should
+
```
[the remark this answers is not in the document]
Comment (ArcAster) [reply docs/video-miners/core-concepts/winning-ticket-concepts.md]: removed

File: docs/video-miners/core-concepts/winning-ticket-concepts.md (hunk 3 above)
[the remark this answers is not in the document]
Comment (ArcAster) [reply docs/video-miners/core-concepts/winning-ticket-concepts.md:35]: Good suggestion, thanks for catching that.

File: docs/video-miners/core-concepts/winning-ticket-concepts.md [hunk 6]
```diff
@@ -0,0 +1,79 @@
...
+- What are tickets and how do they work
+- How adjusting orchestrator attributes affecting tickets impact orchestator payouts
+- How orchestrator behavior changes with incoming tickets
+
+***Related documentation*** [Earnings](https://docs.livepeer.org/video-miners/core-concepts/earnings) and [Payments](https://docs.livepeer.org/video-miners/core-concepts/payments).
+
+## What is a ticket?
+
+Broadcasters using the Livepeer network attach a “ticket” to each segment of a video they want transcoded. A ticket can be thought of as a form of work completed for a given video stream a broadcaster is emitting to the Livepeer network.  The final number of tickets associated with a given stream is a product of the fees proposed by the orchestrator for doing the work and the total length of the video processed.
```
[the remark this answers is not in the document]
Comment (ArcAster) [reply docs/video-miners/core-concepts/winning-ticket-concepts.md:15]: This is meant in a figurative sense, but may be a good point to clarify.  My understanding is that ticketEV is an expectation of the amount of work an O is willing to complete at their given price per pixel - I've found that the exact segment / ticket value relationship is loosely defined in livepeer documentation.

File: docs/video-miners/core-concepts/winning-ticket-concepts.md [hunk 7]
```diff
@@ -0,0 +1,79 @@
...
+
+A ticket is comprised of the following attributes:
+
+- **recipient**: ETH address of recipient (orchestrator completing transcoding work)
+- **faceValue**: The ETH value of the ticket payable to the recipient (orchestrator) if the ticket wins
+- **winProb**: The probability of the ticket winning (number between 0 and 2^256
```
[the remark this answers is not in the document]
Comment (ArcAster) [reply docs/video-miners/core-concepts/winning-ticket-concepts.md:21]: My understanding is that this is adjusted based on how many tickets you're receiving for a given workload, which is loosely based on ticketEV and your price per pixel.  For instance, assuming pixel price stays the same, more tickets -> lower winProb, fewer tickets -> higher win prob - overall probability is about the same for a given unit of transcoding work (relative to stake etc).

File: docs/video-miners/core-concepts/winning-ticket-concepts.md (hunk 3 above)
Comment (ArcAster) [reply docs/video-miners/core-concepts/winning-ticket-concepts.md:35]: I abbreviated to not get too far into the weeds, however it could be a good idea to introduce more complexity in a later revision
